src/components/DatePicker/DatePicker.stories.js

```diff
@@ -127,7 +126,7 @@ import React, { useState } from 'react';
 import 'react-dates/initialize';
 import moment from 'moment';
 import { SingleDatePicker } from 'react-dates';
-import Input from '../Input';
+import {Input} from '@wfp/ui';
```

Comment:
Is the old `import Input from '../Input'` not working here?

Reply:
it does, let me revert that. Thanks for sighting that